### ISSUE TITLE
Specify version of sha.js

### ISSUE THREAD

**Issue by garfik** [dsebastien/webpack-sha-hash#4]
Sha.js project was updated to 2.4.7, which added new dependency hash-base that throw error

```
 51% 448/648 build modules D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\hash-base\index.js:42
  if (this._finalized) throw new Error('Digest already called')
                       ^

Error: Digest already called
    at SHA512.HashBase.update (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\hash-base\index.js:42:30
)    at Compilation.<anonymous> (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack-sha-hash\dist\w
ebpack_sha_hash.js:69:63)    at Compilation.applyPlugins (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\tapable\lib\Tapable.js
:26:37)    at Compilation.createHash (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack\lib\Compilation.
js:791:8)    at Compilation.<anonymous> (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack\lib\Compilation
.js:554:8)    at Compilation.applyPluginsAsync (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\tapable\lib\Tapab
le.js:60:69)    at Compilation.seal (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack\lib\Compilation.js:525:7)    at Compiler.<anonymous> (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack\lib\Compiler.js:39
7:15) at D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\tapable\lib\Tapable.js:103:11 at Compilation.<anonymous> (D:\TeamCity.agents\buildAgent4\work\f2ba92264f7337d\node_modules\webpack\lib\Compilation
.js:445:10)
```

sha.js 2.4.5 work's great

Can you specify version? Thx :)

**Comment by dsebastien** [comment]
Hey there,

Don't hesitate to send a pull request for this :)

**Comment by garfik** [comment]
Sure ;)

**Comment by dsebastien** [comment]
Mh

**Comment by dsebastien** [comment]
I've published 1.0.1 on npm
